gammapy/image/models/new.py

```diff
@@ -42,12 +41,6 @@ def __str__(self):
             ss += '\n\t'.join(covar.pformat())
         return ss
 
-    @staticmethod
-    @abc.abstractmethod
```

Comment:
Why did you remove this? Isn't the point of having an abstract base class that you define an API that the daughter classes have to follow?